packages/cli/src/commands/dev/index.ts

```diff
@@ -28,7 +32,7 @@ export async function runDevServer(entryFile: string = 'src/main.ts') {
       '/@vite/client': {
         injectQuery: (id: string) => id,
         createHotContext,
-        updateStyle() {},
+        updateStyle() { },
```

Comment:
It's weird you have these changes, they should be taken care of by prettier

Reply:
I added a lint script and ran it to revert stylistic changes

Reply:
I have the same question, what do you think of using `ESlint` instead of `Prettier` ? e.g. https://github.com/antfu/eslint-config ,It can handle vue,js,ts,json file well.